docs/api/11_invoices/update-invoice-status.mdx

```diff
@@ -0,0 +1,194 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Update invoice status
+
+## Route
+
+```curl title="PUT"
+/api/v1/invoices/:lago_id
+```
+
+## Usage
+
+<Tabs groupId="prog-language">
+  <TabItem value="curl" label="Curl" default>
+
+  ```bash
+  LAGO_URL="https://api.getlago.com"
+  API_KEY="__YOUR_API_KEY__"
+
+  curl --location --request PUT "$LAGO_URL/api/v1/invoices/5eb02857-a71e-4ea2-bcf9-57d3a41bc6ba" \
+    --header "Authorization: Bearer $API_KEY" \
+    --header 'Content-Type: application/json' \
+    --data-raw '{
+      "invoice": {
+        "status": "succeeded"
+      }
+    }'
+  ```
+  </TabItem>
+
+  <TabItem value="python" label="Python">
+
+  ```python
+  from lago_python_client import Client
+  from lago_python_client.models import InvoiceStatusChange
+
+  client = Client(api_key='__YOUR_API_KEY__')
+
+  status_change = InvoiceStatusChange(
+    lago_id="5eb02857-a71e-4ea2-bcf9-57d8885990ba",
+    status="succeeded"
+  )
+
+  client.invoices().update(status_change)
+  ```
+  </TabItem>
+
+  <TabItem value="ruby" label="Ruby">
+
+  ```ruby
+  require 'lago-ruby-client'
+
+  client = Lago::Api::Client.new({api_key: '__YOUR_API_KEY__'})
+
+  client.invoices.update({
+      lago_id: "5eb02857-a71e-4ea2-bcf9-57d3a41bc6ba",
+      status: "code"
```

Comment:
I believe it should be `status: "succeeded"`

Reply:
Thanks for the catch, updated